### ISSUE TITLE
Search box always searches globaly

### ISSUE THREAD

**Issue by individual-it** [owncloud/web#980]
### Steps to reproduce
1. go into a subfolder
2. search there

### Expected behaviour
only results from within that folder should be listed and/or results from other folder should be listed in a separate section like its done in coreUI 

![image](https://user-images.githubusercontent.com/2425577/56181715-750c1900-602e-11e9-97a9-a7e96f383f2a.png)

### Actual behaviour
all files are listed in the same list and the user has no way to see in which subfolder the file lives. Files with the same name from different folders cannot be distinguished
![image](https://user-images.githubusercontent.com/2425577/56181769-bef4ff00-602e-11e9-9770-1e901d95118e.png)

**Comment by DeepDiver1975** [comment]
phoenix uses a different search api - https://doc.owncloud.com/server/developer_manual/webdav_api/search/search_files.html

Results are always global - for local filtering there is the filter on the right top.

Maybe this is something we first get properly sorted out from the UX and PM perspective.
@michaelstingl @pmaier1

**Comment by DeepDiver1975** [comment]
To be fixed right away:
- show location of the file in the file list - this was the old logic:
https://github.com/owncloud/phoenix/blob/1452bdbc3b6f3dceef19d9f7631b8800cd1c7458/apps/files/src/components/FileList.vue#L47-L64

**Comment by rpocklin** [comment]
I just came across this too, and agree it's not intuitive.  There are no filters available now in the top right in the current build of phoenix, and without it showing the path to the file in the results, making the default option to filter based on the current folder would make more sense (perhaps with a checkbox option to permit a global search as an option).

I have implemented this logic with minimal changes using
`filesSearched = filesSearched.filter(filterFunction)` where `filterFunction` is `file => file.name.startsWith(currentFolder)` and  `currentFolder` is a computed getter of `this.currentFolder.path` passed into the search as an extra parameter (optionally).

If you would like this to be added let me know and I can create a PR if it's likely to be accepted.
